server/app/lib/actions/fusor/deployment/open_shift/install_image.rb

```diff
@@ -0,0 +1,102 @@
+            _("Transfer RHEL Guest 7.2 image to Virtualization Environment")
+          end
+
+          def plan(deployment, image_file_name)
+            super(deployment)
+            plan_self(deployment_id:   deployment.id,
+                      image_file_name: image_file_name)
+          end
+
+          def run
+            ::Fusor.log.debug "================ InstallImage run method ===================="
+            begin
+
+              deployment = ::Fusor::Deployment.find(input[:deployment_id])
+
+              from_image_path    = "/usr/share/#{input[:image_file_name]}/#{input[:image_file_name]}*/"
+              from_tar_file_path = "/tmp"
+              to_img_file_path   = "/root"
+              tar_file_name      = "#{input[:image_file_name]}.tar.gz"
+
+              tar_cmd = "cd  #{from_image_path} && "\
+                        "tar cfvz  /#{from_tar_file_path}/#{tar_file_name} ./* && "\
+                        "cd -"
+
+
+              status, output = run_command(tar_cmd)
+              if status != 0
+                fail _("Unable to create image tar file: Status: %{status}. Output: %{output}") % { :status => status,
+                                                                                                    :output => output }
+              end
+              ::Fusor.log.debug "================ Created the #{input[:image_file_name]} tar file ===================="
+
+              scp_image_file(deployment, "#{from_tar_file_path}/#{tar_file_name}", to_img_file_path)
+              ::Fusor.log.debug "================ Completed Scp #{from_tar_file_path}/#{tar_file_name} to rhevm host! ===================="
+
+              rhevm_host = deployment.rhev_engine_host.facts['ipaddress']
+              ssh_user = "root"
+              ssh_password = deployment.rhev_root_password
+
+              @io = StringIO.new
+              client = Utils::Fusor::SSHConnection.new(rhevm_host, ssh_user, ssh_password)
+              mkdir_cmd = "mkdir -p /#{to_img_file_path}/#{input[:image_file_name]}"
+              client.execute(mkdir_cmd, @io)
+
+              # close the stringio at the end
+              @io.close unless @io.closed?
+
+              @io = StringIO.new
+              untar_cmd = "tar xfvz /#{to_img_file_path}/#{tar_file_name} -C /#{to_img_file_path}/#{input[:image_file_name]}"
+              client.execute(untar_cmd, @io)
+
+              # close the stringio at the end
+              @io.close unless @io.closed?
+
+            rescue Exception => e
+              @io.close if @io && !@io.closed?
+              fail _("Failed to install the RHEL 7.2 image rpm for OSE VMs. Error message: #{e.message}")
+            end
+            ::Fusor.log.debug "================ Leaving InstallImage run method ===================="
+          end
+
+          private
+
+          def scp_image_file(deployment, image_file, image_path)
+            # scp the tar file over to the rhevm host, assume root user
```

Comment:
Unsure if these can be common since this is processed differently than CFME.

They both do SCP a file over, but this one currently needs extra stuff to get the files into a tar so it can be sent.

We plan to change this in a bit when we get rel-eng to produce these files as ova